memory-db/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [dependencies]
 heapsize = { version = "0.4", optional = true }
-parity-util-mem = { version = "0.6", default-features = false }
+parity-util-mem = { version = "0.7.0", default-features = false }
```

Comment:
for the record, this was a breaking change :)

Reply:
I actually knew it was, I totally abused your absence to sneak it in. ;)